cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -49,6 +50,8 @@ func findMyself(client *clientset.Clientset) (*v1.Node, error) {
 }
 
 func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error {
+	validate.WaitForNode(client)
```

Comment:
Hmm, I'm also sceptical whether that will actually work. It still waits for the node to be ready